backend/src/test/java/gov/cdc/usds/simplereport/api/BaseApiTest.java

```diff
@@ -125,6 +130,9 @@ public void setup() {
     _oktaRepo.reset();
     useOrgUser();
     _initService.initAll();
+    when(_addressValidation.getValidatedAddress(any()))
+    .thenReturn(_dataFactory.getAddress());
+    _initService.initCurrentUser();
```

Comment:
were tests failing without this?

Reply:
there seems to be an issue here. I was able to remove `_initService.initCurrentUser();` However,
```
    _initService.initAll();
    when(_addressValidation.getValidatedAddress(any()))
    .thenReturn(_dataFactory.getAddress());
```
calls `getValidatedAddress` where as 
```
    when(_addressValidation.getValidatedAddress(any()))
    .thenReturn(_dataFactory.getAddress());
    _initService.initAll();
```
calls the mocked function

Reply:
*scratches head*. i don't see where in the `initAll()` flow `_addressValidation.getValidatedAddress()` get called.